src/components/about-us/BoardMembers.tsx

```diff
@@ -1,5 +1,5 @@
 import { FC } from 'react'
-import boardSrc from '../../images/about-us/board.jpg'
+const boardSrc = require('images/about-us/board.jpg')
```

Comment:
Let's not mix `require()` and `import`. This codebase uses ES modules, so we should stick with `import`.